app/controllers/ExperimentController.php

```diff
@@ -94,7 +93,17 @@ public function createSubmit()
 
         } else if (isset($_POST['save']) || isset($_POST['launch'])) {
             try {
-                $expId = ExperimentUtilities::create_experiment();
+                $computeResourceId = Input::get("crId");
+                //Validate entered queue details
+                $queueValues = array("queueName" => Input::get("queue-name"),
+                    "nodeCount" => Input::get("nodeCount"),
```

Comment:
I'm pretty sure these aren't the right names for the form fields. Should be `node-count`, `cpu-count` and `walltime-count`. See [experiment-queue-block.blade.php](https://github.com/apache/airavata-php-gateway/blob/develop/app/views/partials/experiment-queue-block.blade.php#L80-L98)

Reply:
Sorry Marcus. I changed this after I tested it on my local machine but I didn’t commit the changes to the PR. Do you want me to do it now?